### ISSUE TITLE
Zamknąc projekt

### ISSUE THREAD

**Issue by mik-laj** [ferrante/otwartasiec#19]
Szanowni, 

Projekt wydaje się martwy? Może warto oznaczyć, że nie już nie funkcjonuje w pliku README.md i skasować linki do strony otwartasiec.pl. Aktualnie pod adresem otwartasiec.pl znajduja się materiały pornograficzne, 

Z wyraźami szacunku

**Comment by ferrante** [comment]
Tak, projekt nie zyje od kilku lat. :( Nie wiedziałem, że ktoś wstawił tam porno, eh.

**Comment by jpacanowski** [comment]
Co się stało z tym projektem? Dlaczego taki super pomysł nie funkcjonuje? W czymś mogę pomóc aby ten projekt zaistniał w sieci?
